### ISSUE TITLE
Write training script

### ISSUE THREAD

**Issue by jstanco** [oceanhackweek/ohw21-proj-drone-turtles#4]
We aim create a training script that uses the `Keras` deep learning framework: https://keras.io/api/

This will require creating functionality specific to training in Keras:
- [ ] Define model class as implementation of `keras.model.Model`.
- [ ] Define data loader as implementation of `keras.utils.Sequence`.
- [x] Define set of `keras.callback.Callback` instances to provide logging/saving functionality during training. These could be custom, or simply take advantage of existing Keras functionality.
- [x] Provide convenient command-line interface (CLI) to specify different training parameters and locations for saving logs and checkpoints.
- [x] Write main training script that makes calls to `model.compile` and `model.fit`.
